tests/characterization_data/filters_expected/sample_nsxv.nsx

```diff
@@ -1,54 +1,612 @@
-<!--
- $Id:$
- $Date:$
- $Revision:$
--->
+<?xml version="1.0" ?>
```

Comment:
Actually, the document created by xml.dom.minidom.parseString() has the xml header by default. I think xml.dom.minidom.toprettyxml does not have the option of removing the xml header.
Does the xml header cause a problem? Please let me know.